resources/ibm-credentials-container.env

```diff
@@ -0,0 +1,9 @@
+SERVICE_1_AUTH_TYPE=container
+SERVICE_1_CR_TOKEN_FILENAME=crtoken.txt
+SERVICE_1_IAM_PROFILE_NAME=iam-user1
+SERVICE_1_IAM_PROFILE_ID=iam-id1
+SERVICE_1_AUTH_URL=https://iamhost/iam/api
+SERVICE_1_SCOPE=scope1
+SERVICE_1_CLIENT_ID=iam-client1
+SERVICE_1_CLIENT_SECRET=iam-secret1
+SERVICE_1_AUTH_DISABLE_SSL=true
```

Comment:
Also, try a test with AUTH_DISABLE_SSL set explicitly to false (see my comments above re: the `__construct_authenticator` function).